hasher-matcher-actioner/hmalib/lambdas/custodian.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+import os
+import random
+import string
+from hmalib.common.logging import get_logger
+from hmalib.common.timebucketizer import TimeBucketizer
+from hmalib.common.models.pipeline import HashRecord
+
+logger = get_logger(__name__)
+
+BUCKET_WIDTH = datetime.timedelta(minutes=10)
+
+
+def lambda_handler(event, context):
+    """
+    Squash the records of multiple timebucketizers into a single file
+    """
+
+    TimeBucketizer.squash_content(
+        datetime.datetime.now(), "hasher", "/tmp/makethisdirectory/", BUCKET_WIDTH
```

Comment:
Make the path argument ("/tmp/..." as an environment variable.)

eg. at line 12: `DURABLE_FS_PATH = os.getenv("DURABLE_FS_PATH", None)`

Only call squash_content if it is defined. If None, log and move on.

Reply:
Extract "hasher" to a constant too and make it a list we'd iterate over. There may be other types in the future.